frontend/packages/dev-console/src/components/pipelines/PipelineRow.tsx

```diff
@@ -22,7 +22,7 @@ interface PipelineRowProps {
 
 const PipelineRow: React.FC<PipelineRowProps> = ({ obj, index, key, style }) => {
   const menuActions = [
-    startPipeline(obj, obj.latestRun),
+    startPipeline(obj, obj.latestRun, 'pipelineruns'),
```

Comment:
@andrewballantyne I had the same doubt, I cleared it up again with serena. For 4.2 let's redirect to pipeline runs and based on the feedback we can change it later.

Reply:
i suggested to behave the same way regardless of where you start, and then get feedback.  This will allow users to learn about the PLR visualization.